eZ/Publish/Core/Repository/ContentService.php

```diff
@@ -1567,6 +1570,62 @@ public function publishVersion(APIVersionInfo $versionInfo)
         return $content;
     }
 
+    /**
+     * @param \eZ\Publish\API\Repository\Values\Content\VersionInfo $versionInfo
+     * @param array $translations
+     *
+     * @throws \eZ\Publish\API\Repository\Exceptions\BadStateException
+     * @throws \eZ\Publish\API\Repository\Exceptions\ContentFieldValidationException
+     * @throws \eZ\Publish\API\Repository\Exceptions\ContentValidationException
+     * @throws \eZ\Publish\API\Repository\Exceptions\InvalidArgumentException
+     * @throws \eZ\Publish\API\Repository\Exceptions\NotFoundException
+     * @throws \eZ\Publish\API\Repository\Exceptions\UnauthorizedException
+     */
+    protected function copyTranslationsFromPublishedVersion(APIVersionInfo $versionInfo, array $translations = []): void
+    {
+        $contendId = $versionInfo->contentInfo->id;
+
+        $publishedContent = $this->internalLoadContent($contendId);
```

Comment:
what if there's no published version? Should the last draft be treated the same?